shadertoy/NodeEditor/PipelineEditor.cpp

```diff
@@ -1579,18 +1579,27 @@ void PipelineEditor::loadFromShaderToy(const std::string& path) {
         return &texture;
     };
     std::unordered_set<std::string> passIds;
+    std::unordered_set<int> dynamicCubeMapChannels;
     const auto isDynamicCubeMap = [&](nlohmann::json& tex) {
         const auto id = tex.at("id").get<std::string>();
-        return passIds.count(id) != 0;
+        return passIds.count(id) != 0 || dynamicCubeMapChannels.count(tex.value("channel", -1)) != 0;
     };
     std::string common;
     for(auto& pass : renderPasses) {
         if(pass.at("name").get<std::string>().empty()) {
             pass.at("name") = generateUniqueName(pass.at("type").get<std::string>());
         }
+
+        if( !pass.at("outputs").empty() && pass.at("type").get<std::string>() == "cubemap") {
```

Comment:
```suggestion
        if(!pass.at("outputs").empty() && pass.at("type").get<std::string>() == "cubemap") {
```
Use clang-format before pushing.